grow/testing/testdata/extract_localized/unlocalized_pod/content/unlocalized_collection/unlocalized_doc_with_localized_parts.yaml

```diff
@@ -1,5 +1,6 @@
----
+#---
 foo@: Unlocalized base doc part in unlocalized collection in unlocalized pöd
----
-$locale: ja
-foo@: Localized doc part in unlocalized doc in unlocalized collection in unlocalized pöd
+#---
+## Since $locales: ja is not specified anywhere, this fails to extract.
+#$locale: ja
+#foo@: Localized doc part in unlocalized doc in unlocalized collection in unlocalized pöd
```

Comment:
Good catch -- will remove that tomorrow AM. Let me know if you find
anything else and I'll give it another pass myself.

On Mon, Aug 1, 2016 at 12:13 AM, Stu Cox <notifications@github.com> wrote:

> In
> grow/testing/testdata/extract_localized/unlocalized_pod/content/unlocalized_collection/unlocalized_doc_with_localized_parts.yaml
> <https://github.com/grow/grow/pull/239#discussion_r72932090>:
>
> >  foo@: Unlocalized base doc part in unlocalized collection in unlocalized pöd
> > ----
> > -$locale: ja
> > -foo@: Localized doc part in unlocalized doc in unlocalized collection in unlocalized pöd
> > +#---
> > +## Since $locales: ja is not specified anywhere, this fails to extract.
> > +#$locale: ja
> > +#foo@: Localized doc part in unlocalized doc in unlocalized collection in unlocalized pöd
>
> Might as well just remove this?
>
> Don’t think we need to test that a commented out string isn’t extracted
> (it doesn’t appear anywhere else).
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/grow/grow/pull/239/files/437cb35792295f0540cdd87a209ebc09b86fc6d0..29b3ff45d34f723b71974fe9583af371e446ce23#r72932090>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAndfQqx2G57hRFwTVdoGyU_8UVZgLFSks5qbZyGgaJpZM4I9hYU>
> .
>



-- 
jeremydw